src/main/java/de/tum/in/www1/artemis/config/SecurityConfiguration.java

```diff
@@ -124,51 +124,51 @@ protected void configure(HttpSecurity http) throws Exception {
         // @formatter:off
         http
             .csrf()
-            .disable()
+                .disable()
```

Comment:
please revert changes in this file as they are not necessary for this PR

Reply:
Done.